olive/systems/utils.py

```diff
@@ -2,66 +2,21 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
-import argparse
```

Comment:
Curious what `misc` means?

Reply:
the file contains multiple utils and I could not find a better name for the file so I choose to misc.py